docs/articles/documentation/extending-testcafe/custom-browser-provider-plugin/index.md

```diff
@@ -0,0 +1,200 @@
+---
+layout: docs
+title: Custom Browser Provider Plugin
+permalink: /documentation/extending-testcafe/custom-browser-provider-plugin/
+---
+# Custom Browser Provider Plugin
+
+TestCafe has built-in support for testing in a number of popular browsers (see [Using Browsers](../../using-testcafe/common-concepts/using-browsers.md)).
+If you need to use a different browser (for example, a browser from a cloud service), you can create a **custom browser provider plugin**.
+The provider should expose methods for performing common actions on the browser windows during testing: opening and closing it, taking screenshot, etc.
+
+To create a browser provider plugin, go through the following steps.
```

Comment:
No, we did it in person. Anyway, it was already discussed and implemented. There is nothing we will do for now. If you have some **real** scenarios that can't be implemented with current approach feel free to develop and share with others your own proposal on how we can extend plugin system.